src/spatial/2d.js

```diff
@@ -81,6 +82,14 @@ Crafty.c("2D", {
      * @see ._attr
      */
     _z: 0,
+
+    /**@
+     * #._globalZ
+     * @comp 2D
+     * When two entities overlap, the one with the larger `_globalZ` will be on top of the other.
+     */
+    _globalZ: null,
```

Comment:
`_globalZ` is only needed for display stuff, can this be moved to `Renderable` too?